lib/middleware/api-authentication-required.js

```diff
@@ -42,8 +44,10 @@ module.exports = function(req, res, next) {
           return res.status(401).json({ error: 'Invalid API credentials.' });
         }
 
+        res.locals.apiKey = authResult;
```

Comment:
Indeed, the type of `authResult` is `AuthenticationResult`, but its `href` is something like:
`'https://api.stormpath.com/v1/apiKeys/ABCDEF123456789'`, so I was confused when naming the property.

I will do as you said. You can close this PR, I will issue a fresh new one on top of the master branch.